ci-operator/step-registry/cucushift/installer/check/private/cucushift-installer-check-private-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ -z "${PUBLISH}" ] || [[ "${PUBLISH}" != "Internal" ]]; then
+    echo "Default publish policy in use, nothing to do. " && exit 0
+fi
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+publish_policy=$(oc get configmap -n kube-system cluster-config-v1 -oyaml | grep 'publish:')
```

Comment:
hmm, I think the key checkpoing of a private cluster need to check if public resources are created on this cluster, an example step on azure - `cucushift-installer-check-azure-fullyprivate`.